setting-up-webauthn-mfa.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Setting up WebAuthn / Passkey multi-factor authentication
+url: /setting-up-webauthn-mfa
+previous: /setting-up-multifactor-authentication
+next: /setting-up-otp-mfa
+---
+
+<em class="t-gray">Setting up WebAuthn / Passkey multifactor authentication</em>
+
+Using WebAuthn, also called Passkey, for multifactor authentication (MFA) is
+the best way to protect your account from takeover. It's stronger and easier
+to use than [OTP codes](/setting-up-otp-mfa).
+
+## Prerequisite
+
+To use WebAuthn, you will need at least _one_ of the following:
+
+* A hardware security token (sometimes called a security key), such as a
```

Comment:
Should we also mention touch-id and windows hello here, since most people have those built in